client/my-sites/plugins-wpcom/jetpack-plugin-item.jsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { identity } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import PurchaseButton from './purchase-button';
+
+export const JetpackPluginItem = ( {
+	plugin,
+	siteSlug,
+	translate = identity
+} ) => {
+	const translatePlan = {
+		premium: translate( 'Premium' ),
+		business: translate( 'Business' ),
+	};
+	const planClasses = [
+		'button',
+		'is-compact',
+		'is-borderless',
+		`plugins-wpcom__is-${ plugin.plan }-plugin`,
+	].join( ' ' );
+	const featureLink = plugin.feature ? `?feature=${ plugin.feature }` : '';
+
+	return (
+		<CompactCard className="plugins-wpcom__jetpack-plugin-item">
+			<a href={ plugin.link } className="plugins-wpcom__plugin-link">
+				<div className="plugins-wpcom__plugin-name">
+					{ plugin.name }
+					{ [ 'premium', 'business' ].includes( plugin.plan ) &&
```

Comment:
`Array.prototype.includes` is not polyfilled and so this line will fail on IE11. Seeing this on errors dashboard for `/plugins/:siteId` path:

```
TypeError: undefined is not a function (evaluating '["premium","business"].includes(plugin.plan)')
```